docs/howto/obsolete-term.md

```diff
@@ -47,17 +47,19 @@ Consult the wiki documentation for suggestions on standard comments:
       
      - [http://wiki.geneontology.org/index.php/Editor_Guide](http://wiki.geneontology.org/index.php/Editor_Guide)
  
-7. If the obsoleted term was replaced by another term in the ontology: In the 'Annotations' window, select ```+``` to add an annotation. In the resulting menu, select ```term replaced by``` and enter the ID of the replacement term.  
+7. If the obsoleted term was replaced by another term in the ontology: In the 'Annotations' window, select ```+``` to add an annotation. In the resulting menu, select ```term replaced by``` and enter the ID of the replacement term (in CURIE format).  
  
 8. If the obsoleted term was not replaced by another term in the ontology, but there are existing terms that might be appropriate for annotation, add those term IDs in the 'consider' tag: In the 'Annotations' window, select ```+``` to add an annotation. In the resulting menu, select ```consider``` and enter the ID of the replacement term.  
 
-9. Save changes. 
+9. Add a link to the GitHub ticket that requested the obsoletion. In the 'Annotations' window, select ```+``` to add an annotation. In the resulting menu, select seeAlso and paste a link to the GitHub ticket.
```

Comment:
do we want the official documentation to recommend using seeAlso or [IAO:0000233](http://purl.obolibrary.org/obo/IAO_0000233) term tracker?
@cmungall @balhoff @matentzn